### ISSUE TITLE
[WCAG] Skip to main content link doesn't work

### ISSUE THREAD

**Issue by spencerwahl** [ramp4-pcar4/story-ramp#164]
The `Skip to main content` link at the top of the `canada.ca` WET templates doesn't do anything currently, the ID it anchors to might be missing. It also looks like the link removes the route parameters from the URL so that might need adjusting if possible.
